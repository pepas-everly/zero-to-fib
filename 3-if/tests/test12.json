../../2-lists/tests/test12.json