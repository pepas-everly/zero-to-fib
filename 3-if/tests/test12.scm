../../2-lists/tests/test12.scm