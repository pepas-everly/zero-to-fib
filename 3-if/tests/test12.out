../../2-lists/tests/test12.out